python/foglamp/services/core/api/certificate_store.py

```diff
@@ -0,0 +1,156 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+import os
+from aiohttp import web
+from foglamp.common import logger
+from foglamp.services.core import connect
+from foglamp.common.configuration_manager import ConfigurationManager
+
+__author__ = "Ashish Jabble"
+__copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+
+_FOGLAMP_DATA = os.getenv("FOGLAMP_DATA", default=None)
+_FOGLAMP_ROOT = os.getenv("FOGLAMP_ROOT", default='/usr/local/foglamp')
+
+
+_logger = logger.setup(__name__, level=20)
+
+_help = """
+    -------------------------------------------------------------------------------
+    | POST             | /foglamp/certificate                                     |
+    | DELETE           | /foglamp/certificate/{name}                              |
+    -------------------------------------------------------------------------------
+"""
+
+
+async def upload(request):
+    """ Upload a certificate
+
+    :Example:
+        curl -F "key=@filename.key" -F "cert=@filename.cert" http://localhost:8081/foglamp/certificate
+        curl -F "key=@filename.key" -F "cert=@filename.cert" -F "overwrite=1" http://localhost:8081/foglamp/certificate
```

Comment:
`"overwrite=1" should be with -F ?`

Reply:
yupe as we are posting `FORM`, so -F for every field=value